components/core/tools/scripts/lib_install/mariadb-connector-c.sh

```
@@ -37,10 +37,11 @@ if [ $installed -eq 0 ] ; then
 fi
 
 echo "Checking for elevated privileges..."
-privileged_command_prefix=""
+install_cmd_args=()
 if [ ${EUID:-$(id -u)} -ne 0 ] ; then
   sudo echo "Script can elevate privileges."
-  privileged_command_prefix="${privileged_command_prefix} sudo"
+  privileged_command_prefix="sudo"
+  install_cmd_args+=(${privileged_command_prefix})
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)